kernel/queue.c

```diff
@@ -199,6 +199,8 @@ static void *k_queue_poll(struct k_queue *queue, s32_t timeout)
 {
 	struct k_poll_event event;
 	int err;
+	unsigned int key;
```

Comment:
u32_t ?

Reply:
I think unsigned int is correct. At least that's what irq_lock() is defined to return and irq_unlock() to take as its parameter.